ext/fs/30_fs.js

```diff
@@ -369,7 +372,7 @@ async function link(oldpath, newpath) {
 
 function toUnixTimeFromEpoch(value) {
   if (ObjectPrototypeIsPrototypeOf(DatePrototype, value)) {
-    const time = value.valueOf();
+    const time = DatePrototypeGetTime(value);
```

Comment:
`Date#valueOf` and `Date#getTime` return the same value. Either is acceptable, but I chose the latter for clarity.